plugin/restore/mysql/mysql.go

```diff
@@ -680,6 +680,221 @@ func (r *Restore) GetSortedBinlogFilesMetaOnServer(ctx context.Context) ([]Binlo
 	return sortBinlogFiles(binlogFiles), nil
 }
 
+// ConvertTsToLocalBinlogCoordinate converts a timestamp to binlog coordinate using local binlog files.
+// We do all kinds of comparison like finding proper backup using the converted BinlogInfo, but not the original timestamp.
+func (r *Restore) ConvertTsToLocalBinlogCoordinate(ctx context.Context, targetTs int64) (*api.BinlogInfo, error) {
```

Comment:
ConvertTsToLocalBinlogCoordinate -> GetBinlogCoordinateByTs

- We're not doing conversion here, and it's typical to name functions with Get().
- "Local" is the implementation details that we don't need to include in the name.

Reply:
SG.